### PR TITLE
fix: set node max mem to 4096

### PR DESCRIPTION
**Pull request by yisiliu**

### CODE REVIEW

**Comment by yisiliu** [comment]
> Does it really resolve the problem?

I was referring to https://github.com/diegomura/react-pdf/issues/666#issuecomment-513799775. One thing you can try is to merge this branch to your other branches and see if everything goes well. I will test it on dependabot's PRs.

File: scripts/ci-build.ts [hunk 1]
```diff
@@ -21,6 +21,7 @@ function buildTypes(name: string): string[] {
 const branch = git.branchName()
 const types = buildTypes(branch.toLowerCase())
 console.log(`Branch: ${branch}`)
+exec('Export', ['NODE_OPTIONS=--max_old_space_size=4096'])  // Increate mem
```
Comment (septs) [inline scripts/ci-build.ts]:
```suggestion
process.env.NODE_OPTIONS = '--max_old_space_size=4096' // Increate mem
```

**Comment by yisiliu** [comment]
it seems to be working: https://app.circleci.com/pipelines/github/DimensionDev/Maskbook/4315/workflows/190c058f-5eae-4019-a572-8a9c1e421e25/jobs/14220

File: scripts/ci-build.ts (hunk 1 above)
Comment (yisiliu) [reply scripts/ci-build.ts]: Why is it labeled outdated?

Comment (yisiliu) [reply scripts/ci-build.ts]: Let me try it first

Comment (yisiliu) [reply scripts/ci-build.ts]: nuh I see..... NVM

**Comment by yisiliu** [comment]
See also: https://support.circleci.com/hc/en-us/articles/360009208393-How-can-I-increase-the-max-memory-for-Node-

**Pull request merged**